README.md

```diff
@@ -1,35 +1,42 @@
-Description
-===========
+# Nano Cookbook
+
+[![Build Status](https://travis-ci.org/chef-brigade/activemq.svg?branch=master)](https://travis-ci.org/chef-brigade/nano-cookbook) [![Cookbook Version](https://img.shields.io/cookbook/v/nano-cookbook.svg)](https://supermarket.chef.io/cookbooks/nano)
 
 Installs nano.
 
-Requirements
-============
+## Requirements
+
+### Platform:
+
+- Ubuntu/Debian
+- RHEL
+- Fedora
+- SLES / openSUSE
+- ArchLinux
```

Comment:
I know RHEL includes Scientific, but it makes no harm to add it. Also, CentOS, Amazon Linux and Oracle Linux are missing in this list. I comment this just in case you think it's appropriate to add them. It's not very important anyway.